### ISSUE TITLE
Numismatics indexer should skip a record on error

### ISSUE THREAD

**Issue by hackartisan** [pulibrary/bibdata#1407]
/concern/numismatics/coins/01bd50f2-3680-4009-81ea-743bef224bc2/orangelight is giving a 500 because of some kind of typeerror (see below). The indexer should log the error and keep going.

```
{
	"id": "AQAAAXnx5TFAnNoKqwAAAABBWG54NVRmcEFBRGVjbUhMNlhJVGR3QVU",
	"content": {
		"timestamp": "2021-06-09T17:50:00.000Z",
		"tags": [
			"source:ruby",
			"service:figgy",
			"filename:production.log",
			"sourcecategory:sourcecode",
			"application:figgy",
			"environment:production",
			"type:webserver"
		],
		"host": "figgy1",
		"service": "figgy",
		"message": "[500] GET /concern/numismatics/coins/01bd50f2-3680-4009-81ea-743bef224bc2/orangelight (Numismatics::CoinsController#orangelight)",
		"attributes": {
			"http": {
				"status_code": 500,
				"url_details": {
					"path": "/concern/numismatics/coins/01bd50f2-3680-4009-81ea-743bef224bc2/orangelight"
				},
				"method": "GET"
			},
			"format": "json",
			"action": "orangelight",
			"controller": "Numismatics::CoinsController",
			"params": {
				"id": "01bd50f2-3680-4009-81ea-743bef224bc2",
				"format": "json"
			},
			"duration": 38100000,
			"sql_queries": "'Sequel::Postgres::Database (0.54) SELECT NULL\nSequel::Postgres::Database (0.29) SELECT NULL\nSequel::Postgres::Database (2.41) SELECT * FROM \"orm_resources\" WHERE (\"id\" = '01bd50f2-3680-4009-81ea-743bef224bc2') LIMIT 1\nSequel::Postgres::Database (0.47) SELECT NULL\nSequel::Postgres::Database (0.29) SELECT NULL\nSequel::Postgres::Database (11.48)           SELECT * FROM orm_resources WHERE\n          metadata @> '{\"member_ids\":[{\"id\":\"01bd50f2-3680-4009-81ea-743bef224bc2\"}]}'\n\nSequel::Postgres::Database (0.46) SELECT NULL\nSequel::Postgres::Database (0.3) SELECT NULL\nSequel::Postgres::Database (0.26) SELECT NULL\nSequel::Postgres::Database (12.5)           SELECT DISTINCT member.* FROM orm_resources a,\n          jsonb_array_elements(a.metadata->'numismatic_place_id') AS b(member)\n          JOIN orm_resources member ON (b.member->>'id')::uuid = member.id WHERE a.id = 'd27c4953-5e21-48b3-b398-854d9e07a8da'\n'",
			"ddsource": [
				"ruby"
			],
			"@version": "1",
			"@timestamp": "2021-06-09T17:50:00Z",
			"db": 29,
			"error": "TypeError: no implicit conversion of nil into Array",
			"sql_queries_count": 10,
			"view": 0
		}
	}
}
```

**Comment by christinach** [comment]
closed by #1408

**Comment by eliotjordan** [comment]
See: https://github.com/pulibrary/figgy/issues/3732
